game/src/MusicConductor/MusicConductor.tscn

```diff
@@ -4,6 +4,8 @@
 
 [node name="MusicConductor" type="Node"]
 script = ExtResource("1_56t1b")
+musicDir = "res://audio/music/"
+firstSongName = "The_Crown.mp3"
```

Comment:
With SFX, I know that extensions are dropped so you can refer to individual sounds without their extension, I'd ask for it here for consistencies sake but its quite a small nitpick.